### ISSUE TITLE
How to use DenseLoss in Pytorch

### ISSUE THREAD

**Issue by spree1010** [SteiMi/denseweight#2]
Hi, I am very interested with this algorithm and want to have a try with DenseLoss in Pytorch,  is there some sample code?

Thank you very much!
Kun

**Comment by SteiMi** [comment]
Hi Kun,

You can use it when you define a custom loss function, for example. You can take a look [here](https://github.com/SteiMi/density-based-weighting-for-imbalanced-regression/blob/main/exp1_and_2/generic_lightning.py) (starting at line 123) for example, where a loss function is defined that basically applies DenseWeights to a simple MSE loss. In this example, DenseWeight's weights are called "relevance" but this "relevance" is basically the same that this package produces.
